src/azure-cli/HISTORY.rst

```diff
@@ -10,6 +10,7 @@ Release History
 **AppConfig**
 
 * Revise help message to exclude unsupported key/label filters.
+* Remove preview tag for most commands excluding managed identity and feature flags.
```

Comment:
from this release, azure cli will automatic generate history.rst based on PR title. no need to manual update history.rst file. instead, pls refine PR title with clear description. thanks.

Reply:
History removed.